src/System.Device.Gpio/System/Device/Pwm/PwmChannel.Linux.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Device.Pwm
+{
+    public partial class PwmChannel
+    {
+        /// <summary>
+        /// Creates a new instance of the <see cref="PwmChannel"/> running on Unix.
+        /// </summary>
+        /// <param name="chip">The PWM chip number.</param>
+        /// <param name="channel">The PWM channel number.</param>
+        /// <param name="frequency">The frequency in hertz.</param>
+        /// <param name="dutyCyclePercentage">The duty cycle percentage represented as a value between 0.0 and 1.0.</param>
+        /// <returns>A PWM channel running on Unix.</returns>
+        public static PwmChannel Create(
+            int chip,
+            int channel,
+            int frequency = 400,
+            double dutyCyclePercentage = 0.5) =>
```

Comment:
I think we should have Create and CreateInternal, Create should have defaults and CreateInternal should be explicit so that we don't accidentally have different defaults on both platforms

Reply:
I think having Create is fine for now since that is what we have in other similar APIs (like I2cDevice.Create or how the GpioController picks a driver), I'll keep it as is and we can discuss later if we want to change to CreateInternal